dist/translation-keys.txt

```diff
@@ -1,60 +1,56 @@
+Outgoing flow count:                                  // traces/sankey/plot.js:143
+Pan                                                   // components/modebar/buttons.js:94
+Produced with Plotly                                  // components/modebar/modebar.js:256
+Reset                                                 // components/modebar/buttons.js:432
+Reset axes                                            // components/modebar/buttons.js:148
+Reset camera to default                               // components/modebar/buttons.js:314
+Reset camera to last save                             // components/modebar/buttons.js:322
+Reset view                                            // components/modebar/buttons.js:583
+Reset views                                           // components/modebar/buttons.js:529
+Show closest data on hover                            // components/modebar/buttons.js:157
+Snapshot succeeded                                    // components/modebar/buttons.js:66
+Sorry there was a problem downloading your snapshot!  // components/modebar/buttons.js:69
+Source:                                               // traces/sankey/plot.js:140
+Taking snapshot - this may take a few seconds         // components/modebar/buttons.js:57
+Target:                                               // traces/sankey/plot.js:141
+Toggle Spike Lines                                    // components/modebar/buttons.js:548
+Toggle show closest data on hover                     // components/modebar/buttons.js:353
+Turntable rotation                                    // components/modebar/buttons.js:288
+Zoom                                                  // components/modebar/buttons.js:85
+Zoom in                                               // components/modebar/buttons.js:121
+Zoom out                                              // components/modebar/buttons.js:130
+kde:                                                  // traces/violin/calc.js:73
+lat:                                                  // traces/scattergeo/calc.js:48
+lon:                                                  // traces/scattergeo/calc.js:49
+lower fence:                                          // traces/box/calc.js:134
+max:                                                  // traces/box/calc.js:132
+mean ± σ:                                             // traces/box/calc.js:133
+mean:                                                 // traces/box/calc.js:133
+median:                                               // traces/box/calc.js:128
+min:                                                  // traces/box/calc.js:129
+q1:                                                   // traces/box/calc.js:130
+q3:                                                   // traces/box/calc.js:131
+trace                                                 // plots/plots.js:439
+upper fence:                                          // traces/box/calc.js:135
```

Comment:
@etpinard @bpostlethwaite thoughts on this format, with the (first) source line for each key included for a translator to use for more context?

Also having all these strings together raises some questions about whether they're all actually what we want, or if any could be made more consistent:
- Is the verb "Double-click" or "Double click"?
- Most of the hover label names (the items ending in `':'`) are lowercase, but some (OHLC, sankey) are capitalized - is this what we want?
- In making this PR I standardized the `"Click to enter <item> title"` ones to have `<item>` capitalized (previously one or two of them were lowercase) - is that correct?
- Most of the `Lib.notifier` items have no punctuation but some of them do. Should we standardize?

Reply:
> thoughts on this format,

I like this format a lot. Nicely done :ok_hand: 

> Is the verb "Double-click" or "Double click"?

I have no preference.

> Most of the hover label names (the items ending in ':') are lowercase, but some (OHLC, sankey) are capitalized - is this what we want?

I'd vote for standardizing to lower case.

>  I standardized the "Click to enter <item> title" [...] is that correct?

fine by me.

> Most of the Lib.notifier items have no punctuation but some of them do. Should we standardize?

No preference.

Reply:
I made all the hover labels lower case, standardized on "Double-click" (when used as a verb, which is the only way we use it in plotly.js text), and tweaked a few other messages -> https://github.com/plotly/plotly.js/pull/2195/commits/97821e912fe3c1363ddf1426529be7bd6eb5523e